DESCRIPTION

```diff
@@ -41,7 +41,8 @@ Imports:
     r2dii.data,
     rlang (>= 0.1.2),
     tidyr,
-    tidyselect
+    tidyselect,
+    stringr
```

Comment:
You may replace `stringr::replace_all()` with multiple calls to `sub()` -- see my other comment.

Reply:
Done.